### ISSUE TITLE
Update for 2016 but not working properly

### ISSUE THREAD

**Issue by andreasrosenbaum** [tdragonite/iTunesFestivalDownloader#2]
Hello,

I tried to edit the 2015 downloader to actual 2016 festival but it's not working properly
The several .ts files were downloaded and joined to a big .ts file but it's not playable.

Also downloading with downloader the .m3u8 link (from wireshark) gives only a big .mp4 file which looks like drm protected.

Elton John 18.09.2016

http://events.applemusic.com/fps/eu1/vod/amf/20160918/54657/vod/v2/8500_256/prog_index_vod.m3u8

Maybe someone can check whats wrong? Cookie or auth?

[itunes-festival.txt](https://github.com/tdragonite/iTunesFestivalDownloader/files/482613/itunes-festival.txt)

Thanks

BR

**Comment by tdragonite** [comment]
Yes, it's DRM protected this year (like I've already said in #1).
Nothing to do, sorry :-(
